website/docs/reference/resource-configs/dbt_valid_to_current.md

```diff
@@ -0,0 +1,116 @@
+---
+resource_types: [snapshots]
+description: "Snapshot dbt_valid_to_current custom date"
+datatype: "{<dictionary>}"
+default_value: {NULL}
+id: "dbt_valid_to_current"
+---
+
+Available in 1.9 or with [Versionless](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless) dbt Cloud.
+
+<File name='snapshots/schema.yml'>
+
+```yaml
+snapshots:
+  my_project:
+    +dbt_valid_to_current: "to_date('9999-12-31')"
+
+```
+
+</File>
+
+<File name='snapshots/<filename>.sql'>
+
+```sql
+{{
+    config(
+        unique_key='id',
+        strategy='timestamp',
+        updated_at='updated_at',
+        dbt_valid_to_current='to_date('9999-12-31')'
+    )
+}}
+```
+
+</File>
+
+<File name='dbt_project.yml'>
+
+```yml
+snapshots:
+  [<resource-path>](/reference/resource-configs/resource-path):
+    +dbt_valid_to_current: "to_date('9999-12-31')"
+```
+
+</File>
+
+## Description
+
+Use the `dbt_valid_to_current` config to set a custom indicator for the value of `dbt_valid_to` in current snapshot records (like a future date). By default, this value is `NULL`. When set, dbt will use this specified value instead of `NULL` for `dbt_valid_to` in the snapshot table.
```

Comment:
```suggestion
Use the `dbt_valid_to_current` config to set a custom indicator for the value of `dbt_valid_to` in current snapshot records (like a future date). By default, this value is `NULL`. When set, dbt will use this specified value instead of `NULL` for `dbt_valid_to` for current records in the snapshot table.
```